src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/SignatureCallingConventionEx.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Reflection.Metadata;
+
+namespace System.Reflection.Emit
+{
+    /// <summary>
+    /// Extensions to the public <cref>System.Reflection.Metadata.SignatureCallingConvention</cref> enum.
+    /// </summary>
+    [Flags]
+    internal enum SignatureCallingConventionEx : byte
```

Comment:
This may be overkill but more descriptive than a const with `0x20`.

Reply:
Can you use the existing `SignatureAttributes.Instance` instead? It means the same thing.

Reply:
Sure ORing with CallConventions makes sense